### ISSUE TITLE
Send Email Error Cannot set "Location"

### ISSUE THREAD

**Issue by kendi95** [emailjs-com/emailjs-react-native#1]
Hi, I tried to send email following the same example availale on doc (https://www.emailjs.com/docs/examples/react-native/) for React Native, but this error is issue.
Someone help me?

My machine configuration:
Intel® Core™ i3-8145U CPU @ 2.10GHz × 4
RAM: 12GB
OS: Ubuntu 22.04.4 LTS

development config:
"expo": "~50.0.14",
"react-native": "0.73.6",

**Comment by xr0master** [comment]
Most likely you are using the wrong package. For the reach native is
@emailjs/react-native

P.S Sorry for missing your request and not responding in time.

**Comment by LuisCabantac** [comment]
Did you managed to fix the issue?

**Comment by xr0master** [comment]
You need to install dependency packages. Please check the docs carefully.

**Comment by LuisCabantac** [comment]
Already installed all the dependencies but still has the same error
```
import emailjs from "@emailjs/react-native";

export async function sendEmail(templateParams: {
  to_email: string;
  to_name: string;
}) {
  const verification = await generateVerificationToken(templateParams.to_email);

  await emailjs.send(
    process.env.EXPO_PUBLIC_EMAILJS_SERVICE_ID!,
    process.env.EXPO_PUBLIC_EMAILJS_TEMPLATE_ID!,
    {
      ...templateParams,
      message: `${process.env.EXPO_WEB_APP_URL}/verify?token=${verification?.value}`,
    },
    { publicKey: process.env.EXPO_PUBLIC_EMAILJS_PUBLIC_KEY! },
  );

  return verification;
}

**Comment by xr0master** [comment]
I cannot reproduce this error. Tried the ios emulator and it works without any issues.

**Comment by LuisCabantac** [comment]
Have you tried with the android emulator?

**Comment by xr0master** [comment]
I did not since the "engine" is the same. But you can debug your app, and let me know if you find any issues.

**Comment by ottosamatori** [comment]
same error, emailJS is not stable...

-> ERROR [NotSupportedError: Cannot set "location".]
-> no changes inside my code

**Comment by xr0master** [comment]
Should be fixed in version 5.1.0
